pkg/client/client.go

```diff
@@ -61,10 +61,10 @@ func (uc *client) resourceInterface(meta object.ObjMetadata) (dynamic.ResourceIn
 	return namespacedClient, nil
 }
 
-// UpdateAnnotation updates the object owning inventory annotation
+// ReplaceOwningInventoryID updates the object owning inventory annotation
 // to the new ID when the owning inventory annotation is either empty or the old ID.
 // It returns if the annotation is updated.
```

Comment:
`// It returns true if the annotation is updated`

Reply:
Good catch. Fixed it